sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py

```diff
@@ -184,6 +188,15 @@ class SystemEventNames(str, Enum, metaclass=CaseInsensitiveEnumMeta):
 
     DataBoxOrderCompletedEventName = 'Microsoft.DataBox.OrderCompleted'
 
+    # spell-checker:ignore MQTT
```

Comment:
is this for cspell ? can it added to the global list?

Reply:
I was doing locally to align with the other spell-checker ignores, do you think it is something that should be global?